util/util.go

```diff
@@ -99,6 +104,31 @@ func GetMachinesForCluster(ctx context.Context, c client.Client, cluster *cluste
 	return &machines, nil
 }
 
+// Internal helper: returns normalized build version (with "v" prefix if needed)
+// If input doesn't match known version pattern, returns empty string.
+// Taken from k8s.io/cmd/kubeadm/app/util
+func NormalizedBuildVersion(version string) string {
```

Comment:
this seems exposed, did you want to make it private?

Reply:
Changing comment. Note that this was direct copied from kubeadm, and then being modified.